db/config.cc

```diff
@@ -984,7 +984,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "The buffer size to use for the buffer-hint feature of the multishard reader when running repair in mixed-shard clusters. This can help the performance of mixed-shard repair (including RBNO). Set to 0 to disable the hint feature altogether.")
     , repair_multishard_reader_enable_read_ahead(this, "repair_multishard_reader_enable_read_ahead", liveness::LiveUpdate, value_status::Used, false,
         "The multishard reader has a read-ahead feature to improve latencies of range-scans. This feature can be detrimental when the multishard reader is used under repair, as is the case in repair in mixed-shard clusters."
-        " This know allows disabling this read-ahead (default), this can help the performance of mixed-shard repair (including RBNO).")
+        " This now allows disabling this read-ahead (default), this can help the performance of mixed-shard repair (including RBNO).")
```

Comment:
I think I wanted to write "knob". The sentence doesn't make sense with "now".

Reply:
I will allow any one letter change